Angular/src/app/messenger-page/left-box/nav-bar/send-message-modal/send-message-modal.component.html

```diff
@@ -0,0 +1,29 @@
+<div tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="exampleModalLabel">New message</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                <form>
+                    <div class="form-group">
+                        <label for="recipient-name" class="col-form-label">Recipient:</label>
+                        <!-- <input type="text" class="form-control" id="recipient-name"> -->
+                        <input type="search" class="form-control" id="recipient-name">
+                    </div>
+                    <div class="form-group">
+                        <label for="message-text" class="col-form-label">Message:</label>
+                        <textarea class="form-control" id="message-text"></textarea>
+                    </div>
+                </form>
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+                <button (click)="sendNewTextMessage()" type="button" class="btn btn-primary">Send message</button>
```

Comment:
Does this work? .ts file for this component is empty. 
Now that we can send messages inside dialog-box component and create conversations by adding friends, maybe this component isn't needed anymore?

Reply:
This modal will be ready used for creating group conversations. Recipients separated by semicolon or sth like that. Not yet implemented